petab/measurements.py

```diff
@@ -200,9 +200,10 @@ def split_parameter_replacement_list(
         delim: delimiter
 
     Returns:
-         List of split values. Numeric values converted to float.
+         List of split values. Numeric values converted to numeric type,
+         parameter IDs as strings.
     """
-    if list_string is None:
+    if list_string is None or list_string == '':
```

Comment:
```suggestion
    if list_string is None or list_string.strip() == '':
```

Reply:
Ah, would need to move after the `isinstance` check below though.

Reply:
That was actually intentional. I would consider blanks there an error.